examples/petset/mongodb-petset-persistent.yaml

```diff
@@ -104,6 +120,7 @@ objects:
           containers:
             - name: mongo-container
               image: "${MONGODB_IMAGE}"
+              imagePullPolicy: IfNotPresent
```

Comment:
I've reminded myself: imagePullPolicy defaults to Always if tag is latest, otherwise IfNotPresent.  Given that the naming convention for this image is mongodb-32-centos7:latest, I happened to find this useful for debugging, but I agree it's not appropriate to check in.  Will revert.